index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
I want to suggest that I choose colors which would be easier to see when playing the game.